### ISSUE TITLE
Midpoint Review - Content - Collaboration Cycle for Accredited Representation Management #65707

### ISSUE THREAD

**Issue by BSmolensky** [department-of-veterans-affairs/va.gov-team#67740]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [x] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts
This a very important process for veterans and your team does a wonderful job of guiding the veteran through it. There are, however, several areas for improvement. 

### MUST

- Please make your heading cases consistent throughout. 
- Page 1, Contact Rep: “help assist” is repetitive. Please simplify the wording.
- Page 5: There are several typos in the “Attorney” paragraph. (1) Please change “as” to “to” in the “…similar services as VSO…” and (2) Please add the word “in” here: “…but usually specialize IN appeals.”
- Page 5: Claims Agent Section - Please change “as” to “to” in the “…similar services as VSO.”
- Page 7: The second sentence underneath the “Appoint Your New Rep” heading is in passive voice. Please put this sentence in active voice. 
- Page 11: “are you the veteran who the claim is being submitted for” is improper English and confusingly worded. Please clean up this sentence. 
- Authorization for Records Access Page: Please fix the grammatical error in the sentence, “By authorizing records access, the representative…” The veteran is the subject of this sentence, not the representative. Please change it to something like, “By authorizing records access, you give the representative…”
- Authorization page: The VFS team needs to provide the user with context here. How will answering the question affect their claim? 
- Terms and Conditions: There are several typos and issues with the Terms and Conditions. (1) Please make the “f” in federal lowercase. (2) “…compensation of whatsoever nature” is confusingly worded. Please clean up this language. (3) “It is understood…” is passive voice. Please reword this sentence to make it active. (4) The final sentence is far too confusingly worded and contains passive voice. Please clean up that sentence. 
- Final Page: “Appointe” is misspelled. Please change it to “Appoint.” 

### SHOULD

- Page 1: The VFS team should include an “in person” option on the very first page. Veterans can go to a regional facility and knock on the doors of the reps who work there. Please change the verbiage to provide the user with accurate info in this regard. 
- Page 3: The veteran should be able to select all three options. Because, perhaps the veteran isn’t sure which option to chose but is more interested in finding someone — anyone — near them who is available. Please update language to reflect this ability. 
- Page 5: The VFS team should change the “will” to “can” or “may” when discussing Attorneys and Claims agents fees. No one is allowed to charge fees during the initial filing period. And some reps don’t charge at all for certain services.
- Page 7: The Appoint a rep page is very confusing. The page begins with “rep selected” but then shows the user their “current rep.”  The page, however, then goes on to say that the user’s new selection has already been saved to their profile. This is further complicated by the fact that the page then goes on to tell the user that they still need to “appoint” their rep. The VFS team should clean up this page to make it easier for the user to understand where they are in the process. 
- Page 8: The expanded dropdown menu underneath “Manually fill out and submit your form” should be updated to reflect the user’s selection. By including all of the forms, the VFS team can confuse the user. 
- Pages 9 - 23: Please change “on” to “after” in the sentence “We’ll save your application on every change.”
- Page 9: The VFS team should make the “gender” category more inclusive. 
- Page 9 and 10: The VFS team should include wording that allows the veteran to understand that they are verifying that their personal info is correct. 
- Authorization page: the VFS team should update the verbiage to reflect when the user is a non-Veteran filling out the application on behalf of a veteran. In this case, the verbiage should be “the veteran” not “my/mine/your.” 
- Authorization page: The list of things the veteran is authorizing (HIV, etc) should be be separated and highlighted instead of imbedded in a long sentence. Bullets would be a good way to go. 
- Final Page: the VFS team should update the “by Mail” section to reflect the exact form the user has already filled out. 

### CONSIDER

- Please consider offering the user an off-ramp much earlier whereby they can simply send the form to the rep so the rep can fill it out. There may be security issues here. 
- Page 1: The VFS team makes the distinction between Claims “or” appeals.  Consider changing that “or” to “And.” Because the representative can assist the veteran with both. 
- Page 5: Please consider adding examples of VSOs — American Legion, VFWs, etc — so that the Veteran can better understand what this option includes. 
- Page 5: VSO can assist with both claims and appeals. Consider updating the language to reflect this. 
- Final Page: Please consider removing Step 2 from “Next Steps” as it was part of the process at the very beginning. To ask the veteran to check availability now seems cruel. 
- Final Page” Please consider deleting the words “with the information you provided.” This simplifies the sentence. 
- Final Page: Consider removing the “by mail” option as the representative is usually the one who submits the form. 

### QUESTIONS
Why would a veteran need a representative to change their address? 

### NOTES
Thank you again for your great work and the opportunity to review it. Please continue to work with CAIA to maximize the effectiveness of your efforts.

**Comment by mtri** [comment]
> Why would a veteran need a representative to change their address?

- Veterans are a transient bunch and can move often and it can be a hassle to update their address every time
- Providing this authorization is just one more way a representative can assist the veteran
- If the Vet doesn't provide authorization for address changes, there isn't any real consequence other than the representative cannot assist the veteran with that process

Source was a conversation with advisors around form 21-22/21-22a [that is recapped here](https://dsva.slack.com/archives/C05L6HSJLHM/p1698180376910259?thread_ts=1698180333.596809&cid=C05L6HSJLHM)

**Comment by oddball-lindsay** [comment]
@aprocik1 has helped us to update the representative definition with the "consider" input, as well as the input of some advisors at the VA that are familiar with the 21-22 process and the Veteran-representative relationship.  There is a paragraph break in the VSO definition to help make the longer content more digestible.  

@BSmolensky - wanted to run this version by you, do you have any follow-up feedback with this updated version? 

> **Veteran Service Organization (VSO)** representatives can help you gather evidence and file your claims, decision reviews, and appeals. They can also communicate with VA about your case on your behalf. Examples of VSOs include the American Legion, County Veteran Service Offices, Disabled American Veterans, and Veterans of Foreign Wars.
> 
> VSO representatives have completed training and passed tests about VA claims and benefits. They provide services to Veterans and their families at no cost.
> 
> **Attorneys** usually work on decision reviews and appeals, including cases that require legal knowledge. They don’t have to take a test about VA claims and benefits, but they must be a member in good standing of the bar association. Attorneys can charge fees for their services.
> 
> **Claims** agents usually work on decision reviews and appeals. They’re independent professionals who have passed a test about VA claims and benefits. Claims agents can charge fees for their services.

**Comment by BSmolensky** [comment]
This is fantastic! Great work. Thank you for running it by me. It looks great: thorough, informative, and not too long. All the info in there is important and clearly stated. Thanks again. Have a great weekend!

**Comment by mtri** [comment]
> Page 3: The veteran should be able to select all three options. Because, perhaps the veteran isn’t sure which option to chose but is more interested in finding someone — anyone — near them who is available. Please update language to reflect this ability.
We will be testing Veteran interaction with this selection in forthcoming usability testing.

> Page 9: The VFS team should make the “gender” category more inclusive.
Gender identity wording is much better in the current profile. Design prototype will be updated to reflect this.

> Final Page: the VFS team should update the “by Mail” section to reflect the exact form the user has already filled out.
We have been told this address listing has been previously researched and all addresses should be included since Veterans can mail to any of the locations regardless of the subject of the claim.

**Comment by mtri** [comment]
Must/should/consider feedback has been incorporated into the prototypes for both Find a representative and for 
Appoint a representative. Thank you for all the very thoughtful feedback!

WIP Find a representative is viewable here (do not open in Sketch/save a new version because these are being used for usability testing currently and may refresh the prototype view. Viewing using the links below is fine though)

[Mobile view](https://www.sketch.com/s/dfb0333d-65db-41f9-807e-6b25d6c5396e/prototype/91C5F671-4BC5-4CE9-B9EA-2D384B20C83A/a/91C5F671-4BC5-4CE9-B9EA-2D384B20C83A) 
[Desktop view ](https://www.sketch.com/s/dfb0333d-65db-41f9-807e-6b25d6c5396e/prototype/100C57E4-87B4-417E-BC4C-F24D85F39C91/a/100C57E4-87B4-417E-BC4C-F24D85F39C91)
[Canvas view](https://www.sketch.com/s/dfb0333d-65db-41f9-807e-6b25d6c5396e#About)

WIP Appoint a representative is viewable here:

[Desktop Prototype view](https://www.sketch.com/s/a75e9329-dae9-4984-867e-451d6e2fa836/prototype/CB1C3151-23CF-40F6-9928-30D7900D7F25/a/CB1C3151-23CF-40F6-9928-30D7900D7F25)
[Mobile Prototype View](https://www.sketch.com/s/a75e9329-dae9-4984-867e-451d6e2fa836/prototype/F68BAB55-E083-411A-A706-CD4251A38CB7/a/F68BAB55-E083-411A-A706-CD4251A38CB7)
[Canvas View](https://www.sketch.com/s/a75e9329-dae9-4984-867e-451d6e2fa836) (Has other states and paths not in prototypes as well as AB test screens for authorization pages.)

As the products were split into three(Find, Appoint, Widget), this feedback has been applied to the newly split products as best as possible.
